src/systems/ec/implementations/react/components/menu/src/MenuItem.jsx

```diff
@@ -25,6 +25,7 @@ export const MenuItem = ({ label, href, isCurrent, subItems }) => {
         data-ecl-menu-link
       >
         {label}
+        {hasSubItems && <span className="ecl-u-sr-only"> (has submenu)</span>}
```

Comment:
the string "(has submenu)" should be translatable, so it has to be put in the data file

Reply:
Please check now